addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -452,13 +452,21 @@ registry.slider = publicWidget.Widget.extend({
         // Initialize carousel and pause if in edit mode.
         this.$target.carousel(this.editableMode ? 'pause' : undefined);
         $(window).on('resize.slider', _.debounce(() => this._computeHeights(), 250));
-        if (this.editableMode) {
-            // Prevent carousel slide to be an history step.
-            this.$target.on('slide.bs.carousel.slider', () => {
-                this.options.wysiwyg.odooEditor.observerUnactive();
-            });
-            this.$target.on('slid.bs.carousel.slider', () => {
-                this.options.wysiwyg.odooEditor.observerActive();
+
+        if (this.editableMode && this.$target[0].matches("section > .carousel")) {
+            this.controlEls = this.$target[0].querySelectorAll(".carousel-control-prev, .carousel-control-next");
+            const indicatorEls = this.$target[0].querySelectorAll(".carousel-indicators > li");
```

Comment:
Why not targeting `[data-slide]` and `[data-slide-to]` elements directly? :+1:

Reply:
Because it is simpler to add back these attributes at destroy (like they are clearly separated so it makes the code specific for each type of controller) 
Also because the "arrows" are the same at the start and destroy, because nothing changes them, but it is not guaranteed to be the case with the indicators (e.g. if we removed or added slides, some indicators will not be there anymore or new ones will be added) so it needs to be "querySelectored" again at destroy => so it was to avoid to re-queryselector the arrows for nothing :slightly_smiling_face: 

But I can change that if you still prefer grouping them :slightly_smiling_face:

Reply:
> But I can change that if you still prefer grouping them 🙂

I am not suggesting to group them but to target them (individually) using `[data-slide]` and `[data-slide-to]` (since if I am not mistaken, the only point is to find the elements that have those attributes to remove them). Targeting something like `.carousel-indicators > li` seems very specific to the current usecase and could thus be broken more easily.

Reply:
Ah my bad (I read what I wanted to read :sweat_smile:) 
But still, ok for when we want to remove the attributes, but for the indicators I can't use `data-slide-to` to retrieve them in the `destroy` since I removed it so I would still need to use `.carousel-indicators > li` at some point
So it's better like this to be consistent no ? :thinking: 
(in the case you agree, should I still use `data-slide` for the arrows ? :slightly_smiling_face: )

Reply:
Mmh indeed, let's forget about this :+1: